site/.eleventy.js

```diff
@@ -1,63 +1,63 @@
+'use strict';
+
 const markdownIt = require('markdown-it');
 const htmlminifier = require('html-minifier-terser');
-const syntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight");
+const syntaxHighlight = require('@11ty/eleventy-plugin-syntaxhighlight');
+
+const htmlminifierConfig = {
+  collapseBooleanAttributes: true,
+  collapseWhitespace: true,
+  conservativeCollapse: false,
+  decodeEntities: true,
+  minifyCSS: true,
+  minifyJS: true,
+  minifyURLs: false,
+  removeAttributeQuotes: true,
+  removeComments: true,
+  removeEmptyAttributes: false,
+  removeOptionalAttributes: true,
+  removeOptionalTags: true,
+  removeRedundantAttributes: true,
+  removeScriptTypeAttributes: true,
+  removeStyleLinkTypeAttributes: true,
+  removeTagWhitespace: false,
+  sortAttributes: true,
+  sortClassName: true,
+};
 
-module.exports = function(eleventyConfig) {
-  eleventyConfig.addPlugin(syntaxHighlight, {
-    templateFormants: ["md", "js", "html"]
-  });
-  eleventyConfig.addPassthroughCopy("src/assets");
-  eleventyConfig.addPassthroughCopy("src/styles");
-  eleventyConfig.addPassthroughCopy("src/script.js");
-  eleventyConfig.addPassthroughCopy("src/site.webmanifest");
+module.exports = eleventyConfig => {
+  eleventyConfig.addPlugin(syntaxHighlight);
 
-  eleventyConfig.addNunjucksFilter("markdown", function(string) {
-    const md = new markdownIt();
+  eleventyConfig.addPassthroughCopy('src/assets');
+  eleventyConfig.addPassthroughCopy('src/styles');
+  eleventyConfig.addPassthroughCopy('src/script.js');
+  eleventyConfig.addPassthroughCopy('src/site.webmanifest');
 
+  eleventyConfig.addNunjucksFilter('markdown', string => {
+    const md = new markdownIt();
     return md.render(string);
   });
 
-  eleventyConfig.addPairedShortcode("markdownConvert", function(content) {
+  eleventyConfig.addPairedShortcode('markdownConvert', content => {
     const md = new markdownIt();
     return md.render(content);
   });
 
-  eleventyConfig.addTransform("htmlminifier", async function(content, outputPath) {
```

Comment:
I also removed `async` from here since it looked wrong, but let me know if you want me to revert it.